src/script/hydra-notify

```diff
@@ -3,24 +3,91 @@
 use strict;
 use utf8;
 use Getopt::Long;
+use HTTP::Server::PSGI;
 use Hydra::Event;
 use Hydra::Event::BuildFinished;
 use Hydra::Helper::AddBuilds;
 use Hydra::Helper::Nix;
 use Hydra::Plugin;
 use Hydra::PostgresListener;
+use Parallel::ForkManager;
+use Prometheus::Tiny::Shared;
+use Time::HiRes qw( gettimeofday tv_interval );
 
 STDERR->autoflush(1);
 STDOUT->autoflush(1);
 binmode STDERR, ":encoding(utf8)";
 
+my $config = getHydraConfig();
+
+my $prom = Prometheus::Tiny::Shared->new;
+# Note: It is very important to pre-declare any metrics before using them.
```

Comment:
Maybe explain why it's important? Though I personally would acquiesce, I could imagine somebody who might ignore this because there's no reason given.

Reply:
:+1: thanks!